DQMOffline/RecoB/python/PrimaryVertexMonitor_cff.py

```diff
@@ -25,3 +25,10 @@
    EtaMax         = cms.double(2.5),
    EtaMin         = cms.double(-2.5)
 )
+
+# same as above, should be in sync with cut used in Vertex finder...
+from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
+from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
+phase1Pixel.toModify(pvMonitor, EtaBin=26, EtaMin=-2.5, EtaMax=2.5)
+phase2_tracker.toModify(pvMonitor, EtaBin=26, EtaMin=-2.5, EtaMax=2.5)
```

Comment:
To my understanding phase2 has eta cut in 4
https://github.com/cms-sw/cmssw/blob/71a61e3cb24ecf0a7647c6e1a1b6871ccf5e4e74/RecoVertex/PrimaryVertexProducer/python/OfflinePrimaryVertices_cfi.py#L57-L60